video/graphics.h

```diff
@@ -69,21 +69,21 @@ char getScreenChar(uint16_t px, uint16_t py) {
 		// Now scan the screen and get the 8 byte pixel representation in charData
 		//
 		for (uint8_t y = 0; y < 8; y++) {
-	    	charRow = 0;
```

Comment:
sorry for the whitespace changes in this PR...  inconsistent formatting bugs me....